pkg/reconciler/pipelinerun/tracing.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2022 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pipelinerun
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/propagation"
+	"go.opentelemetry.io/otel/trace"
+	"knative.dev/pkg/logging"
+)
+
+const (
+	// Tracer is the name of the tracer
+	Tracer = "PipelineRunReconciler"
```

Comment:
Could we name this `TrackerName` or something ? It did confusing me a bit when reading the code (wasn't sure if it was a constant for the name, a global variable, …)

Reply:
`TracerName` rather than `TrackerName`?

Reply:
@afrittoli yes, typo on my part 🙃